src/components/user/repo/registrationPin.repo.js

```diff
@@ -1,33 +1,27 @@
 const { registrationPinModel } = require('../../../services/user/model');
-const { GeneralError } = require('../../../errors');
+const { deleteManyResult } = require('../../helper/repo.helper');
+const { validateEmail } = require('../../helper/uc.helper');
 
+/**
+ * Return pseudonyms for email
+ * @param email
+ */
 const getRegistrationPinsForUser = async (email) => {
+	validateEmail(email);
```

Comment:
why is validation required here? isn't this just a matching of existing entries which could return an empty result? by this i think we must ensure having somehow something a non-empty string but do not require an valid email here, right?

Reply:
this implementation is analogous to another repo implementations. Input parameter is validated before usage. What is the reason to reduce validation here? Is there a valid case when there is a string which is not email?

Reply:
not sure, i would not check it  but instead just to be a non emty string :)

Reply:
ok. as you wish.